sh/tests/solutions/json-researcher.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+curl -s https://content.01-edu.org/assets/superhero/all.json | jq '.[] | select(.id==1)' | grep "name\|\"power\""
```

Comment:
**SUGGESTION** the solution might have to change if you want to implement the exercise with a random id

Reply:
the goal from this exercise is not to teach the student about the script arguments, the script arguments will be teached in the shell scripting quests